lib/format/datetime/formatter.ex

```diff
@@ -598,6 +598,8 @@ defmodule Timex.Format.DateTime.Formatter do
   def format_token(_locale, :us, _date, _modifiers, flags, width) do
     pad_numeric(0, flags, width)
   end
+  def format_token(_locale, :ms, _date = %{microsecond: {us, _precision}}, _modifiers, flags, width), do: pad_numeric(Kernel.round(us / 1000), flags, width_spec(3..3))
```

Comment:
Can you split this on two lines, by adding a newline after the `,` and before `do:`? Same with the next clause